src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -360,6 +360,63 @@ public void getProjectSitePages(@PathVariable String site_id,
 	}
 
 	/**
+	 * REST API call to get CTools site members
+	 * @param request
+	 * @param site_id
+	 * @return
+	 */
+	private HashMap<String, String> get_site_members(
+			HttpServletRequest request, String site_id) {
+		HashMap<String, String> rv = new HashMap<String, String>();
+
+		String membersString = "";
+		String errorMessage = "";
+		String requestUrl = "";
+
+		// login to CTools and get sessionId
+		HashMap<String, Object> sessionAttributes = Utils.login_becomeuser(env,
+				request, Utils.getCurrentUserId(request, env));
+		if (sessionAttributes.containsKey("sessionId")) {
```

Comment:
It would be easier to understand if just return rv immediately when there is no sessionId.